packages/flutter_tools/lib/src/commands/widget_preview.dart

```diff
@@ -96,16 +110,29 @@ final class WidgetPreviewStartCommand extends WidgetPreviewSubCommandBase with C
     required this.fs,
     required this.projectFactory,
     required this.cache,
+    required this.platform,
+    required this.shutdownHooks,
+    required this.os,
   }) {
     addPubOptions();
+    argParser.addFlag(
+      kLaunchPreviewer,
+      defaultsTo: true,
+      help: 'Launches the widget preview environment.',
+    );
```

Comment:
If this flag is for test/debugging the widget preview feature, should we pass `hide: true` here?

Reply:
Done.